wiki/Getting-Started-UIKit.md

```diff
@@ -0,0 +1,239 @@
+
+        saveButton.translatesAutoresizingMaskIntoConstraints = false
+        saveButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        saveButton.topAnchor.constraint(equalTo: emailTextField.bottomAnchor, constant: 24).isActive = true
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    @objc private func savePressed() {
+        proceedInWorkflow(emailTextField.text ?? "")
+    }
+}
+
+// This screen shows an employee only screen
+class SecondViewController: UIWorkflowItem<String, String>, FlowRepresentable {
+    private let email: String
+    private let finishButton = UIButton()
+
+    required init(with email: String) {
+        self.email = email
+        super.init(nibName: nil, bundle: nil)
+
+        view.backgroundColor = .systemGray5
+
+        finishButton.setTitle("Finish", for: .normal)
+        finishButton.setTitleColor(.systemBlue, for: .normal)
+        finishButton.addTarget(self, action: #selector(finishPressed), for: .touchUpInside)
+        finishButton.accessibilityIdentifier = "finish"
+
+        view.addSubview(finishButton)
+
+        finishButton.translatesAutoresizingMaskIntoConstraints = false
+        finishButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        finishButton.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    @objc private func finishPressed() {
+        proceedInWorkflow(email)
+    }
+
+    func shouldLoad() -> Bool {
+        return email.contains("@wwt.com")
+    }
+}
+```
+### Let's talk about what is going on with these view controllers.
+#### **What's this `shouldLoad()`?**
+<details>
+
+It is part of the [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) protocol. It has default implementations created for your convenience but is still implementable if you want to control when a [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) should load in the work flow.  It is called after `init` but before `viewDidLoad()`.
+</details>
+
+## Launching the [Workflow](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/Workflow.html)
+Next, we create a [Workflow](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/Workflow.html) that is initialized with our [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html)s and launch it from a view controller that is already loaded onto the screen (in our case, the 'ViewController' class provided by Xcode).
+
+```swift
+import UIKit
+import Workflow
+
+class ViewController: UIViewController {
+    private let launchButton = UIButton()
+
+    init() {
```

Comment:
would it make sense to change this to the viewDidLoad?

Reply:
In order to allow it to be copied and pasted regardless of whether an individual kept storyboards or fully replaced storyboards then yet, it makes sense to switch that to viewDidLoad().